### PR TITLE
Begining Markdown Converison 

### PR DESCRIPTION
**Pull request by zachChilders**
This adds Intro, ToC, and Injection.  I'll be adding more onto this later today.

### CODE REVIEW

File: 2018/en/0x06-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+## Request for Comments
+- Related vulnerability data to support the project
+- Suggestions and votes of what should be listed in the final OWASP Serverless Top 10 project, including
+any suggested additions not currently on this list
+- Suggestion for “How to Prevent” sections
+- Any additional ​[internal](https://owasp.org)​ and external references that should be included
+
+## Attributions
+Thanks to ​[Protego Labs](https://​protego.io) for sponsoring this report and for everyone else who contributed. Reviewers of this
+report are mentioned on the ​[Acknowledgements page​](page35).
```
Comment (0xbadshah) [inline 2018/en/0x06-release-notes.md]: Page 35 ?

File: 2018/en/0x0a1-injection.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+- Identify trusted sources and resources and whitelist them, if possible
+- For any residual dynamic queries, escape special characters using the specific escape syntax for that
+interpreter
+- Consider all event types and entry points into the system
+- Run functions with the least privileges required to perform the task to reduce attack surface
+- Use a commercial runtime defense solution to protect functions on execution time
+
+## Example Attack Scenario I
+The following function code, repeatedly found in the wild, deserializes data using the eval() function:
+<image>
```
Comment (0xbadshah) [inline 2018/en/0x0a1-injection.md]: Where's the image ?

File: 2018/en/0x06-release-notes.md (hunk 1 above)
Comment (4ppsec) [reply 2018/en/0x06-release-notes.md]: @zachChilders - can you fix the referral to the relevant .md file?

File: 2018/en/0x0a1-injection.md (hunk 2 above)
Comment (4ppsec) [reply 2018/en/0x0a1-injection.md]: @zachChilders - can you add the photos from the /images folder?

**Review by 0xbadshah — COMMENTED** [review]
Why have you added all the files again ?

**Review by 0xbadshah — CHANGES_REQUESTED** [review]
**Merging this PR would add duplicate markdown files**. I just realized that the naming convention for the files are of the format `0x0a1-<topic>.md`. I have already added the files with the format `0xa1-<topic>.md`. (according to OWASP Top 10 file naming convention)

Also I see that images are not included in any of the files.

File: 2018/en/0x00-toc.md [hunk 3]
```diff
@@ -1,36 +1,35 @@
...
+[Release Notes](0x06-release-notes.md)
 
-Table of Contents 3
+[Table of Contents](0x00-toc.md)
 
-Intro: Welcome to Serverless Security 6
+[Intro: Welcome to Serverless Security](0x05-introduction.md)
 
-A1:2017 Injection 7
+[A1:2017 Injection](0x0a1-injection.md)
```
Comment (0xbadshah) [inline 2018/en/0x00-toc.md]: The file should be `0xa1-injection.md`

File: 2018/en/0xa1-injection.md [hunk 4]
```diff
@@ -63,9 +63,8 @@ To exploit this vulnerability, the attacker needed to:
 - Send it to a destination held by the attacker
 
 As a result of the Lambda execution, a request is sent to the attacker, containing the function’s code:
-<image>
-<image>
+![injection-4](images/0x01injection-4.png)
+![injection-5](images/0x01injection-5.png)
 
 ## Serverless Risk Meter
```
Comment (0xbadshah) [inline 2018/en/0xa1-injection.md]: Serverless Risk meter is not required

**Comment by 0xbadshah** [comment]
Currently the images for injection is of the format `0x01injection-<number>.png`. Please change it to `0x01-injection-<number>.png` (with a hyphen separating 0x01 and injection). All other images you have included has followed this naming convention. :+1:

**Comment by 0xbadshah** [comment]
@nu11p0inter I would suggest to squash the commits when merging this PR as there are lots of additions and removals of markdown docs.

File: 2018/en/0x00-toc.md (hunk 3 above)
Comment (zachChilders) [reply 2018/en/0x00-toc.md]: good catch

File: 2018/en/0xa1-injection.md (hunk 4 above)
Comment (zachChilders) [reply 2018/en/0xa1-injection.md]: ok

**Review by 0xbadshah — APPROVED** [review]

**Pull request merged**